components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -141,36 +144,33 @@ public AppMDAO() {
 
     /**
      * This methode is to return a List of existing business owners with out their custom properties.
-     * @param appId
+     * @param businessOwnerId
      * @return
      * @throws AppManagementException
      */
-    public BusinessOwner getBusinessOwner(String appId) throws AppManagementException {
+    public BusinessOwner getBusinessOwner(int businessOwnerId) throws AppManagementException {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5